starters/express-typescript-typeorm-postgres/src/controllers/router.ts

```diff
@@ -1,12 +1,12 @@
-import { Request, Response, Router } from 'express';
+import { NextFunction, Request, Response, Router } from 'express';
 import { TechnologyController } from './technology/technology.controller';
 
 const router = Router();
 
 router.use('/technology', TechnologyController.router);
 
 // TODO: #537 delete this when implementing the health check controller
-router.get('/hello', (req: Request, res: Response) => {
+router.get('/hello', (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Is this change relevant? Because I think a PR will soon be merged that deletes this /hello endpoint and then this change will have a conflict with the target branch.